Twig/Recommendation_Twig_Extension.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Twig;
+
+use eZ\Publish\API\Repository\Repository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Templating\DelegatingEngine;
+
+class Recommendation_Twig_Extension extends \Twig_Extension
```

Comment:
Got it, PSR-2 everywhere... thought I should use Twig code standards, my mistake.